kd/deployment.yaml

```diff
@@ -0,0 +1,255 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hocs-camunda-tools
+  labels:
+    version: {{.VERSION}}
+  annotations:
+    downscaler/uptime: {{.UPTIME_PERIOD}}
+    #downscaler/uptime: "Mon-Fri 08:00-18:00 Europe/London"
+spec:
+  replicas: {{.MIN_REPLICAS}}
```

Comment:
I wonder if we should just hardcode this instead of passing it as a variable? Either way is fine